net/dns/wsl_windows.go

```diff
@@ -156,6 +156,17 @@ func (fs wslFS) Stat(name string) (isRegular bool, err error) {
 	return true, nil
 }
 
+func (fs wslFS) IsSymlink(name string) (bool, error) {
+	err = wslRun(fs.cmd("test", "-L", name))
```

Comment:
I do not have a WSL installation handy to test this behaviour; I either need to set one up or ask someone to test this behaviour.